src/can.c

```diff
@@ -896,6 +919,7 @@ static void thingset_can_thread()
 {
     int err;
 
+    LOG_INF("Initialising ThingSet CAN");
```

Comment:
I prefer not to make the console too noisy and print this only with debug print enabled:
```suggestion
    LOG_DBG("Initialising ThingSet CAN");
```